edgemesh/pkg/server/dns.go

```diff
@@ -108,8 +108,8 @@ func startDnsServer() {
 		if err != nil {
 			continue
 		}
-		for _, q := range que {
-			q.from = from
```

Comment:
q.from = from seems not executed as expected.

Reply:
Good catch, lgtm.  e.g. https://play.golang.org/p/EAMSFaPl9wI   Because the elements of `que` are not pointer. 
cc @arcanique  @kadisi

Reply:
perfect， thanks